### PR TITLE
Nov 2 meeting minutes

### PR DESCRIPTION
**Pull request by mgalexander**
I did add an **acton** for @bengl at the end -- we did not seem to note what I think I recall as an action to pull together a group membership process doc. Please confirm.

### CODE REVIEW

**Comment by mgalexander** [comment]
RE issue #63 for a future ease of linking.

File: meetings/2017-11-02.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  enough
+
+# Actions
+
+* @vdeturckheim to set up meeting with Core triage team to handover HackerOne Organization (https://github.com/nodejs/security-wg/blob/master/processes/security_team_members.md#team-that-triages-security-reports-against-node-core)
+* Discuss issue linked with potential confusion between 2 HackerOne orgs from security reporters
+* @vdeturckheim to create email alias for 3rd party vulnerability report
+* @vdeturckheim to open an issue to open an issue to collect inputs regarding early disclosure. 
+* @mgalexander prepare proposal for bug bounty
+* @bengl agreed to socialize the discussion around security team membership (need to confirm)
```
Comment (bengl) [inline meetings/2017-11-02.md]: This isn't quite what I thought I agreed to. I'll re-listen to the audio and get back to you with a better wording of this.

Comment (mgalexander) [reply meetings/2017-11-02.md]: Can correct the notes to what you intended -- I totally did not mean to sign you up for something other than you were trying to step up to do. I just did not capture it in the moment. I'll amend if you want to just comment it here... thank you for looking!

Comment (bengl) [reply meetings/2017-11-02.md]: `* @bengl agreed to PR security team membership policy` is probably most accurate.

Comment (mgalexander) [reply meetings/2017-11-02.md]: Thank you

File: meetings/2017-11-02.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Arunesh Chandra (@aruneshchandra)-observer
+- Hitesh Kanwathirtha (@digitalinfinity)-observer
+- Adam Baldwin - late (@evilpacket)
+
+# Review of last meeting
+
+
+# New agenda
+
+## Hacker one setup/demo
```
Comment (reedloden) [inline meetings/2017-11-02.md]: HackerOne* here and elsewhere.

File: meetings/2017-11-02.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+# Review of last meeting
+
+
+# New agenda
+
+## HackerOne setup/demo
+
+* @vdeturckheim gave us a walkthrough of what he has setup.
+* Easy to invite people to collaborate on specific vulnerabilities
+*
```
Comment (mhdawson) [inline meetings/2017-11-02.md]: extra * here.

File: meetings/2017-11-02.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+##  Bug Bounty Program https://github.com/nodejs/security-wg/issues/49
+
+* want to add bounties Node.js to list they provide bounties for.
+* not a huge amount of money ($1500 at the top end)
+* does provide some additional motivation for researchers to find/report
+  Issues.
```
Comment (mhdawson) [inline meetings/2017-11-02.md]: Issues -> issues

**Comment by vdeturckheim** [comment]
Is anyone opposed to merging this?

**Pull request merged**